tools/azure-sdk-tools/ci_tools/scenario/generation.py

```diff
@@ -262,7 +264,7 @@ def build_and_install_dev_reqs(file: str, pkg_root: str) -> None:
 
             adjusted_req_lines.append(amended_line)
 
-    adjusted_req_lines = list(map(lambda x: build_whl_for_req(x, pkg_root), adjusted_req_lines))
+    adjusted_req_lines = list(map(lambda x: build_whl_for_req(x, pkg_root, None), adjusted_req_lines))
```

Comment:
This was another place this function was called, not sure what it does but I passed `None` for wheel dir for now.

Reply:
This is right. We've already looked for the package above, so we already know that the prebuilt wheel dir doesn't have it.